tests/ExampleApps/PickingDemoApp/main.cpp

```diff
@@ -0,0 +1,153 @@
+// Include Radium base application and its simple Gui
+#include <Gui/BaseApplication.hpp>
+#include <Gui/Viewer/Viewer.hpp>
+
+// Create dummy scene if needed
+#include <Core/Geometry/MeshPrimitives.hpp>
+#include <Engine/Scene/EntityManager.hpp>
+#include <Engine/Scene/GeometryComponent.hpp>
+#include <Engine/Scene/GeometrySystem.hpp>
+
+// To access render objects
+#include <Engine/Rendering/RenderObject.hpp>
+#include <Engine/Rendering/RenderObjectManager.hpp>
+
+// To access picking results
+#include <Engine/Rendering/Renderer.hpp>
+#include <Gui/RadiumWindow/SimpleWindowFactory.hpp>
+
+#include <Core/Utils/Log.hpp>
+using namespace Ra::Core::Utils; // log
+
+/**
+ * Simple custom windows for picking demonstration
+ */
+class DemoWindow : public Ra::Gui::SimpleWindow
+{
+    Q_OBJECT
+  public:
+    explicit DemoWindow( uint w = 800, uint h = 640, QWidget* parent = nullptr ) :
+        Ra::Gui::SimpleWindow( w, h, parent ) {
+        connect( getViewer(),
+                 &Ra::Gui::Viewer::toggleBrushPicking,
+                 this,
+                 &DemoWindow::toggleCirclePicking );
+        connect(
+            getViewer(), &Ra::Gui::Viewer::rightClickPicking, this, &DemoWindow::handlePicking );
+    }
+    DemoWindow( const DemoWindow& ) = delete;
+    DemoWindow& operator=( const DemoWindow& ) = delete;
+    DemoWindow( SimpleWindow&& )               = delete;
+    DemoWindow& operator=( DemoWindow&& ) = delete;
+    ~DemoWindow() override                = default;
+
+    // activate circle picking
+    void toggleCirclePicking( bool on ) {
+        m_isTracking = on;
+        centralWidget()->setMouseTracking( on );
+    }
+
+    // Handle the picking event generated when clicking on the window with  <RightButton>
+    // When no key is pressed, only the nearest renderObject/Face/vertex of the designated pixel is
+    // selected Once the "C" key is pressed, a orange circle is drawn that define the radius of
+    // selection (<shift+whell modify the radius) This Radius allow to select things for all pixels
+    // within the circle (with the "T", "V" or "E" key pressed while clicking) For each pixel i
+    // within the circle, whe have, for the render object below the mouse ( pickingResult.m_roIdx ):
+    //      pickResults.m_elementIdx : the nearest triangle
+    //      pickResults.m_vertexIdx  : the nearest vertex in the triangle(0, 1 or 2)
+    //      pickResults.m_edgeIdx    : the nearest edge in the triangle
+    void handlePicking( const Ra::Engine::Rendering::Renderer::PickingResult& pickingResult ) {
+        if ( m_isTracking )
+        {
+            std::cout << "Circle selection mode : " << std::endl;
```

Comment:
Is there any reason to not use Radium's LOG instead ?

Reply:
No, we should use Log